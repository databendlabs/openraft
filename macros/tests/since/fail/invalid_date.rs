#[openraft_macros::since(version = "1.0.0", date = "2021-01--")]
fn main() {}
