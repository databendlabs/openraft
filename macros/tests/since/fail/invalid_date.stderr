error: `date`(`2021-01--`) is not valid date string. Expected format: yyyy-mm-dd
 --> tests/since/fail/invalid_date.rs:1:52
  |
1 | #[openraft_macros::since(version = "1.0.0", date = "2021-01--")]
  |                                                    ^^^^^^^^^^^
